steps/server14.js

```diff
@@ -0,0 +1,35 @@
+// Now you should be able to see the message you wrote on the chat printed out in the server console
```

Comment:
The information here is good, but it could be broken up into two shorter steps so we have more time to properly explain how things should work.